include/sound/soc-acpi.h

```diff
@@ -81,7 +81,7 @@ struct snd_soc_acpi_mach {
 	const char *board;
 	struct snd_soc_acpi_mach * (*machine_quirk)(void *arg);
 	const void *quirk_data;
-	void *pdata;
+	const void *pdata;
```

Comment:
NAK. we can't do this, the field is used by other drivers to store non-constant information. We'd have to add a new field but we'll do this later.

Reply:
You also missed the patch 'ASoC: SOF: make ops constant' that I pushed 5 days ago. I missed the spi stuff so will add it through a different PR

Reply:
Good to hear that.